rre-dataset-generator/src/config.py

```
@@ -26,6 +26,8 @@ class Config(BaseModel):
     llm_configuration_file: FilePath = Field(..., description="Path to the LLM configuration file.")
     output_format: Literal['quepid', 'rre']
     output_destination: Path = Field(..., description="Path to save the output dataset.")
+    save_llm_explanation: Optional[bool] = False
+    llm_explanation_destination: Path = Field(..., description="Path to save the llm rating reasoning")
```

Is this working as expected? What if the file is not there? I think the import of the config file should throw an error

---

yes, the file is created because of the llm explanation writer, but the dir `output` is already there.

---

I think maybe
```
output_path = Path(output_path)
os.makedirs(output_path.parent, exist_ok=True)
```
should be added in the method `.write()`, to avoid unexpected errors

---

updated